src/inverseRobot.js

```diff
@@ -7,7 +7,24 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inversedRobot = {};
+
+  for (const key in robot) {
+    inversedRobot[robot[key]] = key;
+  }
+
+  const objectValues = Object.values(robot);
+
+  for (let i = 0; i < objectValues.length; i++) {
+    const firstIndex = objectValues.indexOf(objectValues[i]);
```

Comment:
`objectValues.indexOf(objectValues[i])` is the same as just `i` in this case.

Reply:
wow you are definitely right, i don't know why i didn't think of that 
thank you

Reply:
you are definitely right i don't know why i didn't think of that 😄